### PR TITLE
Add tests that pass for fp16

### PR DESCRIPTION
**Pull request by cnamrata15**
Add tests that pass for fp16 from PR #1562

### CODE REVIEW

**Review by tzerrell — COMMENTED** [review]
LGTM, I only have one inline nitpick

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp [hunk 1]
```diff
@@ -16,39 +16,37 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
     // InferenceEngine::Precision::FP16
 };
 
-const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes = {
```
[the remark this answers is not in the document]
Comment (flaub) [reply plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp:19]: Actually, to make this format nicely, the trick is to add a final `,` to the list.

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp [hunk 2]
```diff
@@ -16,39 +16,37 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
...
+                                                                                    {Tan, {{}}},
+                                                                                    {Elu, {{0.1f}}},
+                                                                                    {Erf, {{}}},
+                                                                                    {HardSigmoid, {{0.2f, 0.5f}}},
+                                                                                    {Selu, {{1.6732f, 1.0507f}}},
+                                                                                    {Ceiling, {{}}},
+                                                                                    {Swish, {{1.0f}}},
+                                                                                    {Mish, {{}}},
+                                                                                    {HSwish, {{}}},
+                                                                                    {SoftPlus, {{}}}};
```
Comment (flaub) [inline plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp:49]: Add a trailing comma here to make it format nicer.

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (tzerrell) [reply plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp:19]: thanks!

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp (hunk 2 above)
Comment (cnamrata15) [reply plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/activation.cpp:49]: Oh cool thanks! I was trying various things and failing

**Review by tzerrell — APPROVED** [review]
LGTM

**Pull request merged**